templates/includes/content.html

```diff
@@ -0,0 +1,5 @@
+<ul>
+    <li>Numbers of drivers: {{ count_drivers }}</li>
+    <li>Number of cars : {{ count_cars }}</li>
+    <li>Number of manufacturers: {{ count_manufacturers }}</li>
+</ul>
```

Comment:
Is good that you know how to use includes, but there is no point to use them in this task. Do it in the content block in `taxi/index.html`.